### PR TITLE
Added Docker multi-arch image support

### PR DESCRIPTION
**Pull request by oneils**
Added docker files to build Docker image for `arm64` that can be run on Raspberry Pi4, Amazon Graviton instances, etc.

`docker-compose-armhf.yml` and `Dockerfile.armhf` are taken as a basis.  The only difference is the replaced image from`arm32v7/alpine:latest` to `alpine:latest` in Dockerfile.

### CODE REVIEW

**Comment by umputun** [comment]
thx for the pr. However, making another dockerfile and another image for arm64 can be not the best solution here. I would rather suggest making a single,  multi-arch image for all 3 architectures. Something like I do [here](https://github.com/umputun/reproxy/blob/master/.github/workflows/ci.yml#L71)

Let me know if you want to make this change, it should be almost trivial.

**Comment by oneils** [comment]
@umputun , thanks for the link. 
I'll make the changes

**Comment by oneils** [comment]
@umputun , do I understant correctly?

I need to update Github Actions to build multi-arch image  and push it  to the Github regestry and Docker registry

**Comment by umputun** [comment]
> @umputun , do I understant correctly?
> 
> I need to update Github Actions to build multi-arch image and push it to the Github regestry and Docker registry

correct. A single Dockerfile and multiarch build with actions is all we need here

**Comment by umputun** [comment]
I have added secrets to allow push to both docker and github registry

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -31,7 +31,54 @@ jobs:
...
+        USERNAME: ${{ github.actor }}
+        GITHUB_SHA: ${{ github.sha}}
+        GITHUB_REF: ${{ github.ref}}
+      run: |
+        ref="$(echo ${GITHUB_REF} | cut -d'/' -f3)"
+        echo GITHUB_REF - $ref
+        echo ${GITHUB_PACKAGE_TOKEN} | docker login ghcr.io -u ${USERNAME} --password-stdin
+        echo ${DOCKER_HUB_TOKEN} | docker login -u ${USERNAME} --password-stdin
+        docker buildx build --push \
+            --build-arg CI=github --build-arg GITHUB_SHA=${GITHUB_SHA} --build-arg GIT_BRANCH=${ref} \
```
Comment (umputun) [inline .github/workflows/ci.yml:61]: pls replace reproxy everywhere in this file to rss2twitter

**Comment by umputun** [comment]
pls let me know as you done and I'll merge it. it would be also nice if you can squash all the commits

**Comment by oneils** [comment]
@umputun , I'm done. 

could you please merge the PR. 

Thanks.

**Review by umputun — APPROVED** [review]
LGTM

**Pull request merged**